modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
 
 Outbound - Slow Requests by Flow::
 The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow. Cell color varies according to the following thresholds:
-
 * Green if the response time is less than 300 ms
 * Yellow if the response time is greater than 300 ms and less than 600 ms
 * Red if the response time is greater than or equal to 600 ms
 
+Inbound - Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```

Comment:
I don't think there should be a period here and elsewhere where there is similar text.

Reply:
```suggestion
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```